src/ui/NoNameFacet/NoNameFacetValues/NoNameFacetValueUtils.ts

```diff
@@ -0,0 +1,23 @@
+import * as Globalize from 'globalize';
+import { INoNameFacetOptions } from '../NoNameFacetOptions';
+import { FacetUtils } from '../../Facet/FacetUtils';
+
+export class NoNameFacetValueUtils {
+  static getFormattedCount(count: number): string {
```

Comment:
I think all of those methods should be directly on the `NoNameFacetValue` class.

I generally do not like "Utils" class like this, they are in normally pretty hard to keep "single role" focused, and often become a "put your code here" kind of file.

Whereas it seems easier to expect that a FacetValue class would be in charge of displaying it's own internal information to the outside world.

Reply:
I agree with Oliver, but maybe there was some other reason to create the Util file? If it's to avoid duplication with the Facet, I think it would be okay, especially if there is an idea to deprecate the original facets?

Reply:
I agree with Olivier, at some point in my implementation it made sense to have an util but not anymore